[2025-02-25T01:27:49.834+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dvc_pipeline.train_model manual__2025-02-25T01:26:11.635563+00:00 [queued]>
[2025-02-25T01:27:49.850+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dvc_pipeline.train_model manual__2025-02-25T01:26:11.635563+00:00 [queued]>
[2025-02-25T01:27:49.851+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-02-25T01:27:49.870+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): train_model> on 2025-02-25 01:26:11.635563+00:00
[2025-02-25T01:27:49.878+0000] {standard_task_runner.py:60} INFO - Started process 182 to run task
[2025-02-25T01:27:49.882+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'dvc_pipeline', 'train_model', 'manual__2025-02-25T01:26:11.635563+00:00', '--job-id', '76', '--raw', '--subdir', 'DAGS_FOLDER/weather_pipeline_dag.py', '--cfg-path', '/tmp/tmpv1i9wd31']
[2025-02-25T01:27:49.886+0000] {standard_task_runner.py:88} INFO - Job 76: Subtask train_model
[2025-02-25T01:27:49.905+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2025-02-25T01:27:49.954+0000] {task_command.py:423} INFO - Running <TaskInstance: dvc_pipeline.train_model manual__2025-02-25T01:26:11.635563+00:00 [running]> on host 7efe327aec71
[2025-02-25T01:27:50.060+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dvc_pipeline' AIRFLOW_CTX_TASK_ID='train_model' AIRFLOW_CTX_EXECUTION_DATE='2025-02-25T01:26:11.635563+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-25T01:26:11.635563+00:00'
[2025-02-25T01:28:55.325+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-02-25T01:28:55.378+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=dvc_pipeline, task_id=train_model, execution_date=20250225T012611, start_date=20250225T012749, end_date=20250225T012855
[2025-02-25T01:28:55.463+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-02-25T01:28:55.523+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
